voxblox_ros/src/transformer.cc

```diff
@@ -112,9 +112,7 @@ bool Transformer::lookupTransformTf(const std::string& from_frame,
   return true;
 }
 
-bool Transformer::lookupTransformQueue(const std::string& from_frame,
-                                       const std::string& to_frame,
-                                       const ros::Time& timestamp,
+bool Transformer::lookupTransformQueue(const ros::Time& timestamp,
```

Comment:
@helenol  this function (which has a duplicate in voxblox node) didn't use the from/to frame names, I guess that was on purpose?

Reply:
Just matching the same interface as the other function.... MIght make sense to keep it that way in case of the future if we actually do smarter resolution of queues? But this is also frickin' unlikely to ever happen to be honest. What do you think?

Reply:
I think we can always extend it again if this ever happens. Keeping them will just confuse people, IMO.